### DISCUSSION TITLE
Enable notification

### DISCUSSION THREAD

**Discussion by Pjjlltt** [Docile-Alligator/Infinity-For-Reddit#954] [Duplicate Ideas] [answered]
On the Reddit webpage on desktop you have the option to enable notification bell for topics when new activity/discussion happens in that thread/subject.

I have only found the option to save a topic but doing that I have to manually keep going back to check if there has been any new activity. 
So I would consider this a big QOL improvement to have in Infinity also if possible.

![Screenshot_20220814-161327_1_1_1](https://user-images.githubusercontent.com/88964484/184541463-df73b37e-24d9-4436-8ba8-5ee60b3b815e.png)

**Comment by Wladefant** [comment, marked as answer]
@Pjolt this is a duplicate of #963.
Please add your idea as a comment to #963
